### PR TITLE
Pulls the entirety of tg onto our codebase to fix shit

### LINKED ISSUE #27817: Flaky hard delete: /mob/living/basic/mining/raptor/green
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9184764149/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: /mob/living/basic/mining/raptor/green hard deleted 1 times out of a total del count of 5 at code/modules/unit_tests/create_and_destroy.dm:99
```

### LINKED ISSUE #27883: Flaky hard delete: /mob/living/basic/raptor/red
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9262905874/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: /mob/living/basic/raptor/red hard deleted 1 times out of a total del count of 5 at code/modules/unit_tests/create_and_destroy.dm:99
```

### LINKED ISSUE #27903: Clarke internal ore box breaks upon picking a boulder up
Clarke was made before the boulder change so its internal ore box is able ot pick up boulders but this bricks the whole mech. because it cant eject boulders its code wasnt designed with that in mind and it was never updated.

### LINKED ISSUE #28261: Multiple errors in flaky test modify_fantasy_variable
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9586408143/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
modify_fantasy_variable: [18:13:07] Runtime in ,: bad icon operation
proc name: New (/icon/New)
src: /icon (/icon)
call stack:
/icon (/icon): New('sound/items/duct_tape_snap.ogg', null, null, null, null)
Held Fish Analyzer, Left (/datum/greyscale_config/fish_analyzer_inhand_left): Generate("#711f86", null)
Greyscale (/datum/controller/subsystem/processing/greyscale): GetColoredIconByType("/datum/greyscale_config/fish_a...", "#711f86")
the fish analyzer (/obj/item/fish_analyzer): update greyscale()
the fish analyzer (/obj/item/fish_analyzer): set greyscale("#711f86", null, null, null, null)
the fish analyzer (/obj/item/fish_analyzer): set greyscale(/list (/list), null, null, null, null)
the fish analyzer (/obj/item/fish_analyzer): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the fish analyzer (/obj/item/fish_analyzer), 0, /list (/list))
the fish analyzer (/obj/item/fish_analyzer): New(0)
/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable): allocate(/obj/item/fish_analyzer (/obj/item/fish_analyzer))
/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable): Run()
RunUnitTest(/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at OUTDATED_TEST:1
modify_fantasy_variable: [18:13:07] Runtime in ,: bad icon operation
proc name: New (/icon/New)
src: /icon (/icon)
call stack:
/icon (/icon): New('sound/items/duct_tape_snap.ogg', null, null, null, null)
Held Fish Analyzer, Left (/datum/greyscale_config/fish_analyzer_inhand_left): Generate("#711f86", null)
Greyscale (/datum/controller/subsystem/processing/greyscale): GetColoredIconByType("/datum/greyscale_config/fish_a...", "#711f86")
the fish analyzer (/obj/item/fish_analyzer): update greyscale()
the fish analyzer (/obj/item/fish_analyzer): Initialize(0)
the fish analyzer (/obj/item/fish_analyzer): Initialize(0)
Atoms (/datum/controller/subsystem/atoms): InitAtom(the fish analyzer (/obj/item/fish_analyzer), 0, /list (/list))
the fish analyzer (/obj/item/fish_analyzer): New(0)
/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable): allocate(/obj/item/fish_analyzer (/obj/item/fish_analyzer))
/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable): Run()
RunUnitTest(/datum/unit_test/modify_fantas... (/datum/unit_test/modify_fantasy_variable), /list (/list))
RunUnitTests()
/datum/callback (/datum/callback): InvokeAsync() at OUTDATED_TEST:1
```

### LINKED ISSUE #28273: Flaky hard delete: /obj/effect/decal/remains/human
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9588479230/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: /obj/effect/decal/remains/human hard deleted 1 times out of a total del count of 4 at code/modules/unit_tests/create_and_destroy.dm:99
```

### LINKED ISSUE #28398: Flaky test orphaned_genturf: Floating genturf (/turf/open/genturf) detected
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9690050620/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
orphaned_genturf: Floating genturf (/turf/open/genturf) detected at (84, 171, 5) : /area/ruin/unpowered. Why was it not replaced? at code/modules/unit_tests/orphaned_genturf.dm:7
```

### LINKED ISSUE #28641: Multiple flaky test failures in create_and_destroy, orphaned_genturf
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/9803383578/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
orphaned_genturf: Floating genturf (/turf/open/genturf) detected at (87, 171, 5) : /area/ruin/unpowered. Why was it not replaced? at code/modules/unit_tests/orphaned_genturf.dm:7
create_and_destroy: /mob/living/basic/butterfly hard deleted 1 times out of a total del count of 19 at code/modules/unit_tests/create_and_destroy.dm:99
```

### LINKED ISSUE #29039: Flaky hard delete: /obj/machinery/power/shieldwallgen
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/10081010101/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: /obj/machinery/power/shieldwallgen hard deleted 1 times out of a total del count of 1 at code/modules/unit_tests/create_and_destroy.dm:99
```

### LINKED ISSUE #29068: Flaky hard delete: /obj/machinery/power/apc
Flaky tests were detected in [this test run](https://github.com/Skyrat-SS13/Skyrat-tg/actions/runs/10106238285/attempts/1). This means that there was a failure that was cleared when the tests were simply restarted.
Failures:
```
create_and_destroy: /obj/machinery/power/apc hard deleted 1 times out of a total del count of 1 at code/modules/unit_tests/create_and_destroy.dm:99
```

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

Someone help me

Includes every PR earlier than #29667

We need to get this working in about a week if we don't want the repo even more fucked up

Fixes every single missed mirror, every small fucky mistake of the past 4 years. Probably introduces more

## How This Contributes To The Skyrat Roleplay Experience

It doesnt

## Proof of Testing

Doesn't even compile yet lol there's like 1500 conflicts left at the time of opening this

### CODE REVIEW

**Comment by nikothedude** [comment]
holy fuck

**Comment by Swiftfeather** [comment]
When the crystal meth be hittin

**Comment by projectkepler-ru** [comment]
stil nowhere close

**Comment by nikothedude** [comment]
I NEED A HERO

**Comment by nikothedude** [comment]
Where have all the good men gone
And where are all the gods?
Where's the streetwise Hercules
To fight the rising odds?
Isn't there a white knight upon a fiery steed?
Late at night, I toss and I turn
And I dream of what I need

**Comment by thegrb93** [comment]
It seems the merge is reverting a lot of skyrat modified code. Not sure how you merged it, but I don't think that should happen.

How are you dividing the work? Maybe I can take a folder

**Comment by Majkl-J** [comment]
> It seems the merge is reverting a lot of skyrat modified code. Not sure how you merged it, but I don't think that should happen.

I picked skyrat changes where there was a skyrat comment, or for documentation files where there was a blame linking to it. There may have been some misses when strange was helping me so if you just point out the files in questions ill take a look. Also this can't really be divided into smaller parts it needs to be one massive chunk of conflict resolution.

As for reversals aside from reverting a lot of the prefcode modifications (It needs a complete rewrite due to a removal of mutant bodyparts on tg) nothing else should've been touched

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 1]
```diff
@@ -412,7 +412,7 @@
 		/obj/item/crowbar/cyborg/power, // Skyrat ADDITION - Combines Crowbar and Wirecutters into one
 		/obj/item/multitool/cyborg, //Skyrat ADDITION - Adds multitool for easier access
 		/obj/item/borg/cyborg_omnitool/engineering,
-		///obj/item/borg/cyborg_omnitool/engineering, //Skyrat REMOVAL
+		/obj/item/borg/cyborg_omnitool/engineering,
```
Comment (thegrb93) [inline code/modules/mob/living/silicon/robot/robot_model.dm]: Here's an example of the merge being weird, or maybe you did this intentionally?

Comment (thegrb93) [inline code/modules/mob/living/silicon/robot/robot_model.dm:415]: Here's an example of the merge being weird

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm [hunk 2]
```diff
@@ -81,9 +81,4 @@
 	var/obj/item/I = user.get_active_held_item()
 	if(!I)
 		return
-	//SKYRAT EDIT START
-	if(istype(I, /obj/item/kinetic_gauntlet))
-		var/obj/item/kinetic_gauntlet/gauntlet = I
-		gauntlet.next_attack = 0 //i hate this but i also don't know how else to implement this
-	//SKYRAT EDIT END
```
Comment (thegrb93) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm]: Here's another one

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 1 above)
Comment (Maia-J) [reply code/modules/mob/living/silicon/robot/robot_model.dm]: Doubt I would make it leave 2 omnitools intentionally so probably some merge fuckery

**Comment by Majkl-J** [comment]
We're almost there

**Comment by Majkl-J** [comment]
image

**Comment by Swiftfeather** [comment]
The madlad actually fixed 3 full years of code repo debt...

**Pull request merged**